web/src/inbox_ui.js

```diff
@@ -948,10 +1095,18 @@ function get_focus_class_for_header() {
 
 function get_focus_class_for_row() {
     let focus_class = ".inbox-left-part";
-    if (col_focus === COLUMNS.UNREAD_COUNT) {
-        focus_class = ".unread_count";
-    } else if (col_focus === COLUMNS.ACTION_MENU) {
-        focus_class = ".inbox-topic-menu";
+    switch (col_focus) {
+        case COLUMNS.UNREAD_COUNT: {
+            focus_class = ".unread_count";
+            break;
+        }
+        case COLUMNS.ACTION_MENU: {
+            focus_class = ".inbox-topic-menu";
+            break;
+        }
+        case COLUMNS.TOPIC_VISIBILITY: {
+            return ".change_visibility_policy";
```

Comment:
is there a reason not to just assign this to `focus_class` and have a single return?

Reply:
Nice catch, linter changed the code and didn't recheck it, my bad.